pkg/utils/resources.go

```diff
@@ -70,3 +77,18 @@ func GetPodsTotalRequestsAndLimits(podList *corev1.PodList) (reqs corev1.Resourc
 	}
 	return
 }
+
+func GetUsedPodNums(podList *corev1.PodList) (res corev1.ResourceList) {
+	podQuantity := resource.Quantity{}
+	res = corev1.ResourceList{}
+	for _, p := range podList.Items {
+		pod := p
```

Comment:
In the loop, A new variable `pod` to store the current Pod, which is unnecessary. should directly use `p`.